services/scan/src/store.ts

```diff
@@ -626,6 +642,7 @@ export class Store {
       batchId
     ) as { count: number };
     this.client.run('delete from batches where id = ?', batchId);
+    this.invalidateBatchesExportedAt();
```

Comment:
Does this really invalidate anything? Any previous backups would still have all the other batches (plus this one). I'm unsure how this ought to work cc @benadida.

Reply:
Yeah, I was unsure about whether deleting a batch should invalidate an export. As mentioned [here](https://github.com/votingworks/vxsuite/pull/1580#discussion_r827341485), if this should not invalidate the backup, we can revert d2fe24b0223dc1277930a73dc0a919dbffaf06bb.